src/PostgREST/Unix.hs

```diff
@@ -1,44 +1,14 @@
 module PostgREST.Unix
-  ( runAppWithSocket
-  , installSignalHandlers
+  ( installSignalHandlers
   ) where
 
-import qualified Network.Socket           as Socket
-import qualified Network.Wai.Handler.Warp as Warp
-import qualified System.Posix.Signals     as Signals
-
-import Network.Wai        (Application)
-import System.Directory   (removeFile)
-import System.IO.Error    (isDoesNotExistError)
-import System.Posix.Files (setFileMode)
-import System.Posix.Types (FileMode)
+import qualified System.Posix.Signals as Signals
 
 import qualified PostgREST.AppState as AppState
 
 import Protolude
 
 
--- | Run the PostgREST application with user defined socket.
-runAppWithSocket :: Warp.Settings -> Application -> FileMode -> FilePath -> IO ()
-runAppWithSocket settings app socketFileMode socketFilePath =
-  bracket createAndBindSocket Socket.close $ \socket -> do
-    Socket.listen socket Socket.maxListenQueue
-    Warp.runSettingsSocket settings socket app
-  where
-    createAndBindSocket = do
-      deleteSocketFileIfExist socketFilePath
-      sock <- Socket.socket Socket.AF_UNIX Socket.Stream Socket.defaultProtocol
```

Comment:
Hm, not that this is gone. How are we handling unix sockets?

Reply:
We do, in https://github.com/PostgREST/postgrest/pull/3034/files#diff-f6074642dc32a94e9e1997b21b0df631b2d3f098e0a81673edc22b89a86f49edR176

Reply:
I've had to this code back in order to be able to handle `server-unix-socket` config on unsupported platforms